### DISCUSSION TITLE
Headerline with breadcrumbs

### DISCUSSION THREAD

**Discussion by gbrlgn** [helix-editor/helix#8465] [Ideas]
I'd like to posit/request a feature akin to Emacs' LSP Mode's breadcrumbs, which are fed by the LSPs to draw indicative sections of the file in the header bar. It helps millions for us ops folks whom have to mess with gynormous YAML files and need to constantly pinpoint where we actually are looking at:

![image](https://github.com/helix-editor/helix/assets/47647695/a053e9a9-1721-40bf-b78c-b2ca1b2d0f85)

**Comment by kirawi** [comment]
Related https://github.com/helix-editor/helix/pull/6118
